README.md

```diff
@@ -51,10 +51,17 @@
     <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
   </ul>
 </details>
+
 ### Key Features <a name="key-features"></a>
-- **[Linters]**
-- **[Webhint]**
-- **[Stylelint]**
+
```

Comment:
- [ ] Your readme is missing the usage section which should describe how the project is to be displayed on the browser kindly add this section to your readme file.
- [ ] since you do not have any information for deployment section I would kindly suggest you remove it from the headers in your readme because currently the `deployment` link is broken as it won't take you to any section of the same name in your readme